api/common.js

```diff
@@ -18,10 +18,7 @@ export const unwrapImage = (image, urlParams) => {
   const imageFile = image && image.fields && image.fields.file
 
   if (!imageFile) {
-    return {
-      url: undefined,
-      title: undefined,
-    }
+    return undefined
```

Comment:
Why is this returning `undefined` wouldn't it be better to return just an empty object ?